go/vt/grpcclient/client.go

```diff
@@ -55,3 +58,23 @@ func SecureDialOption(cert, key, ca, name string) (grpc.DialOption, error) {
 	creds := credentials.NewTLS(config)
 	return grpc.WithTransportCredentials(creds), nil
 }
+
```

Comment:
To avoid every client from calling this, we should move this into grpcclient/client_auth_static.go. Then add an `init` function in that file that registers this function as additional callback that `Dial` has to perform before performing the grpc Dial.

If done this way, all the clients will remain unchanged. If someone wants to remove this form of auth, they just delete the file from the build.

Ping me on slack if you need clarification.